### DISCUSSION TITLE
Help about the new Embedded node options

### DISCUSSION THREAD

**Discussion by ElucGeek** [ZeusLN/zeus#1803] [Q&A] [answered]
Hello,
I'm testing the new v8.0.0 beta 1 and I'm wondering what are doing all these new options in the Embedded node menu. Especially the Peers sub-menu. What is the Neutrino Peers option? Say I have a node (beside the embedded one), should I add my node here? Should I add the olympus one as well?
Same for Zero conf Peers, what does it do if I add my node here?

Other options that would be interesting to understand better, even if already briefly explained:
Express graph Sync: I see it's all disabled. When would I want to enable this? What are the pros and cons of using this?
Pathfinfing: apriori vs bimodal. Pros and cons?

**Comment by Darth-Coin** [comment, marked as answer]
>Say I have a node (beside the embedded one), should I add my node here?

If you run your own Bitcoin core node, yes, we encourage you to use your own node, for the simple fact that will be closer to your location where are you using Zeus and not using a far away neutrino peer.
Here you can read more about how to enable neutrino in your own node: https://docs.lightning.engineering/lightning-network-tools/lnd/enable-neutrino-mode-in-bitcoin-core

>Should I add the olympus one as well?

Yes, as a backup you can have 1-2 more neutrino peers in that list.
Here is a list of vouched neutrino peers that can be used. test each one their ping and select only those that have a lower ping, better under 150-200ms and add them to your list.

btcd1.lnolymp.us | btcd2.lnolymp.us - for US region
sg.lnolymp.us - for Asia region
btcd-mainnet.lightning.computer - for US region
uswest.blixtwallet.com (Seattle) - for US region
europe.blixtwallet.com (Germany) - for EU region
asia.blixtwallet.com - for Asia region
node.eldamar.icu - for US region
noad.sathoarder.com - for US region
bb1.breez.technology | bb2.breez.technology - for US region
neutrino.shock.network - US region

You could search for more peers here, but is not certain that these will be always online.
https://bitnodes.io/nodes/?q=NODE_COMPACT_FILTERS

>Same for Zero conf Peers, what does it do if I add my node here?

0conf peers is another story. Read more here: https://docs.zeusln.app/for-users/embedded-node/trusted-funding.
Also here: https://stacker.news/items/909079

>re Express Graph Sync

You would want it Enabled and also activate the option "reset EGS at startup" in case that you see some of your payments failed for "no route". Also you can switch from different sources for graph sync (Olympus, Blixt, own)
EGS is like downloading a map of active LN nodes and channel from a specific node that laready have it updated. So your mobile node will be faster synced and ready to do payments. This is due to the particularity that your mobiel node is not always online to sync this information.

>Pathfinfing: apriori vs bimodal.

Leave bimodal. Put apriori only when you really have issues with paying a specific destination and it doesn't work.

**Comment by ElucGeek** [comment]
Hi @Darth-Coin , thanks for your answer on my 17 months old question 😅, never too late.
In the meantime I started a new node running CLN, so I cannot using it as neutrino peer. My LND on Umbrel is still running but I closed many channels and plan to stop it completely at some point (thus I figured out it was running much better with only a limited number of large well connected channels, so still use it as a kind of backup for now).

Graph Sync and Path finding are still interesting to understand. I might do an in deep presentation of Zeus at a Bitcoin meetup, these information will be super useful, and might be for others I hope.

**Comment by Darth-Coin** [comment]
Yeah, I was just browsing the discussion section on Zeus github and saw your old post. I thought will be good to answer.
Here I wrote 2 guides about Zeus embedded node, if that could help you more:
- https://darth-coin.github.io/wallets/getting-started-zeus-wallet-en.html
- https://darth-coin.github.io/wallets/zeus-node-advanced-usage-en.html
